opts/opts.go

```diff
@@ -215,6 +215,7 @@ func ValidatePath(val string) (string, error) {
 	return val, nil
 }
 
+// FIXME should it return error as it is always nil ?
```

Comment:
Ok that's kinda what I guessed, it has to be there because it is used as a ``type ValidatorFctType func(val string) (string, error)``. I think I should just remove the comment for now :stuck_out_tongue_closed_eyes:.

Environment and thus Label (that are validated the same way) are always *valid*, just rewritten a bit sometimes.